package.json

```diff
@@ -4,32 +4,45 @@
   "description": "An H5P widget that sets the value of the field to a [UUID](https://en.wikipedia.org/wiki/Universally_unique_identifier). The widget does not render the input field.",
   "main": "h5p-editor-uuid.js",
   "scripts": {
-    "start": "vite build --mode=development --watch --minify=false",
-    "build": "tsc && vite build",
-    "lint": "prettier .",
-    "lint:fix": "prettier . --write"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "NODE_ENV=production webpack",
+    "watch": "webpack --watch",
+    "lint": "eslint \"./**/*.ts\"",
+    "lint:fix": "eslint --fix \"./**/*.ts\""
   },
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/NDLA-H5P/h5p-editor-uuid.git"
+    "url": "git+https://github.com/NDLANO/h5p-editor-uuid.git"
   },
-  "keywords": [],
-  "author": "",
-  "license": "ISC",
+  "keywords": [
+    "h5p",
+    "library"
+  ],
+  "author": "NDLA",
+  "license": "MIT",
   "bugs": {
-    "url": "https://github.com/NDLA-H5P/h5p-editor-uuid/issues"
+    "url": "https://github.com/NDLANO/h5p-editor-uuid/issues"
   },
-  "homepage": "https://github.com/NDLA-H5P/h5p-editor-uuid#readme",
+  "homepage": "https://github.com/NDLANO/h5p-editor-uuid#readme",
   "devDependencies": {
     "@tsconfig/strictest": "^2.0.1",
-    "h5p-types": "^1.17.0",
-    "prettier": "^2.8.8",
-    "prettier-config": "github:boyum/prettier-config",
-    "typescript": "^5.0.4",
-    "vite": "^4.3.3"
+    "@typescript-eslint/eslint-plugin": "^5.59.2",
+    "eslint": "^8.39.0",
+    "eslint-config-standard-with-typescript": "^34.0.1",
+    "eslint-plugin-import": "^2.27.5",
+    "eslint-plugin-n": "^15.7.0",
+    "eslint-plugin-promise": "^6.1.1",
+    "h5p-types": "^1.18.0",
+    "ts-loader": "^9.4.2",
+    "typescript": "^5.0.4"
   },
-  "prettier": "prettier-config",
   "dependencies": {
-    "h5p-utils": "^3.2.1"
+    "@babel/preset-env": "^7.21.5",
```

Comment:
The only "distribution" dependency is `h5p-utils`, the others are `devDependencies`. One does technically not need to differentiate between them, but if one does, this should be sorted correctly.